packages/python/plotly/plotly/tests/test_core/test_graph_objs/test_performance.py

```diff
@@ -0,0 +1,47 @@
+import sys
+import time
+from unittest import TestCase
+import pytest
+import numpy as np
+import plotly.graph_objs as go
+import plotly.io as pio
+from plotly.tests.b64 import b64, _b64
+
+
+def test_performance_b64():
+    rand_arr_1 = np.random.random(100000)
+    rand_arr_2 = np.random.random(100000)
+    raw_arr_1 = rand_arr_1.tolist()
+    raw_arr_2 = rand_arr_2.tolist()
+    b64_arr_1 = b64(rand_arr_1)
+    b64_arr_2 = b64(rand_arr_2)
+
+    # Test the performance of the base64 arrays
+    b64_start = time.time()
+    fig = go.Scatter(x=b64_arr_1, y=b64_arr_2)
+    b64_time_elapsed = time.time() - b64_start
+
+    # Test the performance of the raw arrays
+    raw_start = time.time()
+    fig = go.Scatter(x=raw_arr_1, y=raw_arr_2)
+    raw_time_elapsed = time.time() - raw_start
+
+    # b64 should be faster than raw
+    assert (b64_time_elapsed / raw_time_elapsed) < 0.85
```

Comment:
Could you print out the actual ratio to see if we could further reduce `0.85`?

Reply:
Since the `to_json` return full figure it contains lots of other setup inside `layout.template`.
I think it would be better to compare the difference between those two sizes e.g. 
```b64_time_elapsed -normal_time_elapsed < 12345```

Reply:
I think the ratio might make more sense here because different environments will have different elapsed times, but hopefully they'll be proportionally slower if they're in a slower environment.

Reply:
Yes I wanted this for sizes not performance :)

Reply:
Ah ok will do